Cargo.toml

```diff
@@ -54,6 +54,7 @@ ordermap = { version = "0.5.6", default-features = false, features = [] }
 ipnet = { version = "2.11.0", default-features = false, features = [] }
 iptrie = { version = "0.10.2", default-features = false, features = [] }
 mio = { version = "1.0.3", default-features = false, features = [] }
+nom = { version = "8.0.0", default-features = false, features = [] }
```

Comment:
Would your change work with nom `7.1.3`, to appease CI? Otherwise we can mark the crate as an exception (or see if @daniel-noland has another suggestion).

Reply:
Probably. Let me check!

Reply:
It works fine. We use the nom crate just to provide a hex dump of the buffer.